### DISCUSSION TITLE
Add unified authors data file to reduce header attributes for blog posts.

### DISCUSSION THREAD

**Discussion by Ouch1978** [facebook/docusaurus#5042] [Ideas]
In Jekyll, there's a authors.yml file that stores all authors' profile which includes social media links, author title and avartar url, so that we could just put only author attribute in blog header without author_title, authorURL, author_image_url... in every post.

**Comment by slorber** [comment]
Agree. I suggest posting this to [our Canny board](https://docusaurus.io/feedback/) so that we figure out how much interest there is for this feature.
